ext/SConscript

```diff
@@ -42,40 +44,54 @@ for subdir, extensions, prepFunction in libs:
+    license_files["fmtlib"] = File("#ext/fmt/LICENSE.rst")
     localenv = prep_default(env)
     localenv.Prepend(CPPPATH=Dir('#ext/fmt/include'))
     libraryTargets.extend(
         localenv.SharedObject(multi_glob(localenv, 'fmt/src', 'cc')))
     for name in ('format.h', 'ostream.h', 'printf.h', 'core.h', 'format-inl.h'):
-        build(copyenv.Command("#include/cantera/ext/fmt/" + name,
-                              "#ext/fmt/include/fmt/" + name,
-                              Copy('$TARGET', '$SOURCE')))
+        ext_copies.extend(
+            copyenv.Command("#include/cantera/ext/fmt/" + name,
+                            "#ext/fmt/include/fmt/" + name,
+                            Copy("$TARGET", "$SOURCE"))
+        )
 
 if env['system_sundials'] == 'n':
     localenv = prep_default(env)
     localenv.Prepend(CPPPATH=[Dir('#include/cantera/ext'),
                               Dir('#ext/sundials/src/sundials')])
-    license_files.append(('Sundials', 'sundials/LICENSE'))
+    license_files["Sundials"] = File("#ext/sundials/LICENSE")
 
     # Generate sundials_config.h
     sundials_configh = {}
     if env['OS'] != 'Windows':
         sundials_configh['SUNDIALS_USE_GENERIC_MATH'] = 1
     if env['use_lapack']:
         sundials_configh['SUNDIALS_BLAS_LAPACK'] = 1
-    localenv.AlwaysBuild(env.Command('#include/cantera/ext/sundials/sundials_config.h',
-                                     'sundials_config.h.in',
-                                     ConfigBuilder(sundials_configh)))
+    sundials_configh_build = env.Command("#build/ext/sundials_config.h.build",
+                                         "sundials_config.h.in",
+                                         ConfigBuilder(sundials_configh))
+    # This separate copy operation, which SCons will skip if sundials_config.h.build is
+    # unmodified, prevents unnecessary re-copies of files in the #include directory
+    localenv.AlwaysBuild(sundials_configh_build)
+    ext_copies.extend(
+        localenv.Command("#include/cantera/ext/sundials/sundials_config.h",
+                         "#build/ext/sundials_config.h.build",
+                         Copy("$TARGET", "$SOURCE"))
+    )
 
     # Copy sundials header files into common include directory
     for subdir in ('sundials', 'nvector', 'cvodes', 'ida', 'sunmatrix',
                    'sunlinsol', 'sunnonlinsol'):
         for header in multi_glob(env, 'sundials/include/'+subdir, 'h'):
-            build(copyenv.Command('#include/cantera/ext/%s/%s' % (subdir, header.name),
-                                  '#ext/sundials/include/%s/%s' % (subdir, header.name),
-                                  Copy('$TARGET', '$SOURCE')))
+            ext_copies.extend(
+                copyenv.Command(f"#include/cantera/ext/{subdir}/{header.name}",
+                                f"#ext/sundials/include/{subdir}/{header.name}",
+                                Copy("$TARGET", "$SOURCE"))
+            )
```

Comment:
The `build` function returns the input targets so it can left as a nested call here and then you don't have to fiddle around with making things `Depend` on `ext_copies` later.

Reply:
Sorry, I'm not getting what you're saying... I needed the sdist to explicitly depend on all the files that were copied from `ext` because otherwise it wasn't necessarily building in the right order... For some reason, SCons didn't seem to see that these files were required for the package.